src/mix/api/channels-types.ts

```diff
@@ -21,4 +23,53 @@ export type ChannelsRenameBodyParams = {
   displayName: string;
 }
 
+/** @hidden */
+export type ChannelsCreatePathParams = {
+  /** ID of the relevant project. */
+  projectId: string;
+}
+
+/** @hidden */
+export type ChannelsConfigPathParams = ChannelsCreatePathParams;
+
+// REVIEW: this should almost certainly be put in its own file
+export const ChannelModalities = [
+  'MODE_UNSPECIFIED',
```

Comment:
I just tried creating a channel through Swagger and it reject MODE_UNSPECIFIED. same for COLOR_UNSPECIFIED". So I suggest we remove those as they just complicate things. We will have to check what projects:create does with that list of colors before making that change.